### DISCUSSION TITLE
Do we include compiled files/dependencies in the repo? Potential actions and build steps

### DISCUSSION THREAD

**Discussion by Sephsekla** [WordPress/wp-feature-notifications#99] [General]
- In order to have a downloadable, installable plugin, we need the compiled assets, however this is a pain to track.
- One option discussed was using https://github.com/marketplace/actions/create-release along with a build step (@lorenzosfienti).

**Comment by JasonTheAdams** [comment]
I do not recommend storing compiled assets in the repo. That gets very messy. You're welcome to checkout our Github Actions over at GiveWP.
- Generate a zip on-the-fly from any branch/commit: https://github.com/impress-org/givewp-github-actions/blob/master/.github/workflows/generate-zip.yml
- Release to wp.org: https://github.com/impress-org/givewp-github-actions/blob/master/.github/workflows/wp-org-release.yml

This is our re-usable workflow repo, so it would work slightly different here, but the job steps are the same.
